### ISSUE TITLE
Player hangs while trying to save a snapshot

### ISSUE THREAD

**Issue by elafynoh** [clsid2/mpc-hc#1274]
while trying to use the capture image function the player hangs and never comes back. Most likely the path i have used last time does not exist anymore (change of drives).I haven't found an option to reset the path. Also updating or deinstalling/installing the player does not help.

**Comment by clsid2** [comment]
Use File > Save image, and you can choose the path.

**Comment by elafynoh** [comment]
> Use File > Save image, and you can choose the path.

i did exactly this, but the player then hangs forever before any path selection appears. There is one drive missing on my system while trying this and as soon as i add the drive again the save image option works again. The strange thing is, that if i add the drive again the path i am offered is the default picture path and not a path on the re-installed drive.

**Comment by clsid2** [comment]
When it hangs, do the following:
Start the Windows Task Manager (Ctrl+Shift+Esc).
Right-click on mpc-hc64.exe and choose "Create dump".
Zip/rar/7zip the .dmp file to make it smaller.
Upload it somewhere for me. For example http://www.sendspace.com

Then I can check what is wrong.

Is it an external USB harddrive?

**Comment by clsid2** [comment]
I am unable to reproduce. If I set an invalid path, then MPC-HC shows an error "unable to create file" followed by a prompt to choose a new location.

I think the hang occurs in Windows code, when attempting to access that location. What happens if you try to open the location when pasting that path in the Explorer location bar?

**Comment by elafynoh** [comment]
I have tried to recreate the error too, but unfortunatelly once i plugged in my other harddrives and it started to work again i could remove them again and the file selection works as usual with the default documents picture folder. Previously i had drives C,D,G,H,I,J,K,L as well as Q for bd drive and the error occured. I was pretty sure that the last time i used the image save option it was from a file of an E or F drive.

Then i added the other drives which are then became E nd F and it worked again.

I then played a file from E and save an image to a path in E. I then removed E and F again and tried to play a file from H and safe the image and it also did not crash.

Well, it does seem like an error which only occured once in a very specific situation.

In any case, thank you very much for looking into this. I am using MPC as my default player all the time, so i am happy it works perfectly now.
